ruby/syntax/context_free/expressions/primary/control/BeginExpression.sdf3

```diff
@@ -2,8 +2,13 @@ module BeginExpression
 
 imports
   BodyStatement
+  CompoundStatement
   
 context-free syntax
   BeginExpression.BeginExp = <begin <BStmt> end>
   
+  BeginBodyExpression.BEGINExp = <BEGIN { <CStmt> }>
+  EndBodyExpression.ENDExp = <END { <CStmt> }>
```

Comment:
Begin and End can also be lowercase, just like while and if and case and do et cetera.